explorer/src/routes/provisioners/+page.svelte

```diff
@@ -0,0 +1,28 @@
+<script>
+  import { ProvisionersCard } from "$lib/components";
+  import { duskAPI } from "$lib/services";
+  import { appStore } from "$lib/stores";
+  import { createPollingDataStore } from "$lib/dusk/svelte-stores";
+  import { onDestroy, onMount } from "svelte";
+
+  const pollingDataStore = createPollingDataStore(
+    duskAPI.getProvisioners,
+    $appStore.fetchInterval
+  );
+
+  onMount(() => pollingDataStore.start());
+  onDestroy(pollingDataStore.stop);
+
+  $: ({ data, error, isLoading } = $pollingDataStore);
+  $: ({ isSmallScreen } = $appStore);
+</script>
+
+<section id="blocks">
```

Comment:
Apart from the ID being wrong, do we need an id at all (same for the blocks page component)?